manifests/init.pp

```diff
@@ -136,10 +135,9 @@
 #   Also see [`makestep_seconds`](#makestep_seconds).
 # @param queryhosts
 #   This adds the networks, hosts that are allowed to query the daemon.
-#   Note that `port` needs to be set for this to work.
```

Comment:
The reference is generated by calling `bundle exec rake strings:generate:reference` and it looks like this isn't reflected there.

Reply:
I think that the docstring in init.pp is correct now, neither of init.pp or REFERENCE.md
has that note anymore. Probablu the reference shoult be regenerated anyway?